apps/docs/audit-log/types.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Event Types
+description: 'Available audit log event types'
+---
+
+
+
+## Workspaces
+
+The Workspace actions that will create a new audit log item.
+
+- workspace.create - A Workspace is created.
```

Comment:
Could you wrap all of the event types as inline code please?

And also watch out for capital letters, these should be lower case, no?

```suggestion
- `workspace.create` - A workspace is created.
```